monkey/monkey_island/cc/services/reporting/report.py

```diff
@@ -363,6 +386,12 @@ def process_drupal_exploit(exploit):
         processed_exploit['type'] = 'drupal'
         return processed_exploit
 
+    @staticmethod
+    def process_zerologon_exploit(exploit):
```

Comment:
This file is getting ridiculous. I think we should choose one: either refactor `report.py` or refactor `telems` to DAL right after this PR. This refactoring will make sure that were constantly improving our codebase. @mssalvatore what do you think?

Reply:
@VakarisZ We discussed moving the processors into standalone files so we can later on refactor properly into plugins. :)